### PR TITLE
fix: stored the contributors properly

### PR DESCRIPTION
**Pull request by CorsoGabriele**
This PR Fixes #issue_number

## Please review
- [ ] @libremente

### CODE REVIEW

File: fare/file_management/templates/file_management/create.html [hunk 1]
```diff
@@ -27,6 +27,7 @@ <h2>Create record</h2>
             </div>
             <div class="form-group {{ 'has-error' if form.contributor_name.errors }}">
               <label for="contributor_name">{{ form.contributor_name.label }}</label>
+              <small>(separare i diversi autori con una virgola)</small>
```
Comment (libremente) [inline fare/file_management/templates/file_management/create.html:30]: Here, and in all the other fronted strings, we need to understand what's the best policy for translations in order to stick with them in all the rest of the platfom.
I mean, should we write in English and then translate or not?

**Comment by libremente** [comment]
Is this PR ready for review? @CorsoGabriele

File: fare/file_management/templates/file_management/create.html (hunk 1 above)
Comment (CorsoGabriele) [reply fare/file_management/templates/file_management/create.html:30]: I think that write in English and then translate is the best way, i'll make the chages

File: fare/file_management/views.py [hunk 2]
```diff
@@ -31,8 +31,15 @@ def create():
     form = RecordForm()
     # if the form is submitted and valid
     if form.validate_on_submit():
-        # we creare one contributor object with the submitted name
-        contributors = [dict(name=form.contributor_name.data)]
+        listnames = form.contributor_name.data.split(",")
+        listcontributors = []
```
Comment (libremente) [inline fare/file_management/views.py:35]: Personally, I don't really like this variable names, can we call them "list_names" and "list_contributors"?

**Comment by libremente** [comment]
Can you please rebase @CorsoGabriele

File: fare/file_management/views.py (hunk 2 above)
Comment (CorsoGabriele) [reply fare/file_management/views.py:35]: I'll change it

**Comment by CorsoGabriele** [comment]
rebase done

**Pull request merged**